bats/schema-changes.bats

```diff
@@ -257,6 +248,7 @@ SQL
     dolt add test
     dolt commit -m "committed table so we can see diffs"
     dolt table rm test
+    skip "cannot change primary keys"
```

Comment:
Instead of skipping why not just change the tests to drop table, re-add the table with new primary keys.